spec/greek_script_spec.rb

```diff
@@ -1,6 +1,6 @@
 describe 'Greek script' do
   context 'alpha A α' do
-    it 'α in και' do
+    it 'α in και', pending: 'fixme' do
```

Comment:
This test is failing because ckey 12698843 doesn't have vern_title_245a_display field. The reason it doesn't have this field is because the vernacular script is in the 245 field and not 880. In fact, this record was created in Symphony 7/5/2018, hasn't been modified since, has no order attached or real items, so it could simply be a mistake somebody made creating a record. I'm inclined to have Data Control fix it (remove the record) rather than mark this test pending. Or put a constraint on the test to return only solr docs with date_cataloged?

Reply:
I think both of those options are good ones. I've gone ahead and added a commit that limits the scope of the queries to only "catalogued" items.

Let me know if that makes sense or if there is a different way I should approach this.